src/components/Modals/SaveCourseModal.vue

```diff
@@ -17,19 +17,24 @@
       <!-- create a rectangular border-->
       <div class="saveCourseModal-header-text">
         <span> Collections </span>
-        <button class="saveCourseModal-header-text-addButton" @click="addNewCollection">
+        <button class="saveCourseModal-header-text-addButton" @click="addCollection">
           <img src="@/assets/images/plus.svg" alt="add new collection" />
         </button>
       </div>
     </div>
 
     <div class="saveCourseModal-body">
       <div class="saveCourseModal-body-content">
-        <p>{{ collection }}</p>
-        <!--Must find all possible collections
-            Checkbox Style
-            Need a collections variable in firestore
-        -->
+        <div v-for="collection in collections" :key="collection">
```

Comment:
We would guarantee that collection names are unique then — not tracking internally with IDs and whatnot? (Is a reasonable assumption to make.)

Reply:
I was thinking the same...I think this is a reasonable thing to assume for now.